staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -835,6 +835,17 @@ type CDISpec struct {
 	PriorityClass *CDIPriorityClass `json:"priorityClass,omitempty"`
 }
 
+type ComponentConfig struct {
+	// NodePlacement describes scheduling configuration for specific CDI components
+	NodePlacement sdkapi.NodePlacement `json:"nodePlacement,omitempty"`
```

Comment:
In order to be json compatible, should be this:
```golang
type ComponentConfig struct {
   sdkapi.NodePlacement
  ...
```

Reply:
No `NodePlacement` member.  Want to embed the struct

Reply:
Can you inspect me why my code will causes josn compatible problem? I think json.Marshal and json.Unmarshal has no problem. Although this may be a rudimentary question, looking forward to your answer. : -)